src/app/(layout)/new/routes/page.tsx

```diff
@@ -99,52 +96,12 @@ export default async function Routes({ searchParams }: PageParams) {
 					</TableHeader>
 					<TableBody>
 						{routes.map((r) => (
-							<TableRow key={r.id}>
-								<TableCell>
-									<Tooltip
-										trigger={
-											<span className="font-medium text-black">
-												{r.origin_iata}
-											</span>
-										}
-										children={r.origin_name}
-									/>
-								</TableCell>
-								<TableCell>
-									<Tooltip
-										trigger={
-											<span className="font-medium text-black">
-												{r.destination_iata}
-											</span>
-										}
-										children={r.destination_name}
-									/>{' '}
-								</TableCell>
-								<TableCell>{formatMinutes(r.average_duration)}</TableCell>
-								<TableCell className="space-x-2">
-									{r.aircraft_short_names
-										.split(',')
-										.filter((ac) => aircraft.includes(ac))
-										.map((ac) => (
-											<Badge key={ac} variant="blue">
-												{ac}
-											</Badge>
-										))}
-									{r.aircraft_short_names
-										.split(',')
-										.filter((ac) => !aircraft.includes(ac))
-										.map((ac) => (
-											<Badge key={ac} variant="gray">
-												{ac}
-											</Badge>
-										))}
-								</TableCell>
-							</TableRow>
+							<Row route={r} aircraft={aircraft} />
 						))}
 					</TableBody>
 				</Table>
 				<Pagination totalCount={totalCount} resource="route" />
 			</div>
 		</div>
 	);
-}
+}
```

Comment:
Add newline to end of file

see https://archive.eslint.org/docs/rules/eol-last